src/diagnostics.toit

```diff
@@ -10,16 +10,16 @@ class Diagnostics:
   The time to first fix.
   0, if no fix has been obtained.
   */
-  time_to_first_fix/Duration
+  time-to-first-fix/Duration
```

Comment:
Do kebab changes in a separate PR.
(Note: `toit tools kebabify code */*.toit` does this automatically)